### ISSUE TITLE
The setup is extremely confusing to newbies and either the setup process or the documentation should be improved

### ISSUE THREAD

**Issue by fffffgggg54** [gridcoin-community/Gridcoin-Research#903]
I have seen/heard many instances of newbies trying to set up the wallet and failing, saying it was too confusing. I advise that either the setup procedure be better documented or the setup clarified. I also think that example gridcoinresearch.conf files should be put up for investor mode, pool researching, and solo researching.

**Comment by RoboticMind** [comment]
I agree. I think the documentation should also make clear that a lot of Gridcoin is needed to solo mine.

**Comment by WhiteCat6142** [comment]
To start solo mine beginners need to  "advertise beacon", and it sometimes fails. 
Should be more simple and easy.

**Comment by Firescrat** [comment]
As im new i can tell you wich things annoyed me most.

--> each thing that needs to be put in the conf file should be in the settings! With checkboxes for listener,... textboxes where i can input my email adress, CPID, Boinc install dir, datadir, addnodes button, list of current nodes in the conf, ability to delete the nodes, max connections.... Thats small simple improvements wich help a lot, even if the settings take effect after a restart. Things that need to be written manually in a conf file are ONLY for hardcore users and it techs.

--> Advertise beacon command button in the settings and alongside a notification somewhere if a beacon was sent or needs to be sent (maybe above current poll).

--> standard settings button for solo mining, investor, research in the settings

--> Maybe in the help section a link to the gridcoin wiki

--> Version number in the header, Notification for new Version.

--> Notification if a chainfork is likely and what to do

These simple things would improve usability greatly


I have one other thing wich would be an improvement. I dont want the GridcoinResearch in the hidden Appdir! First why in a hidden dir where noone looks the documents folder would be more likely, second this directory is on my main hdd(sdd) and not changeable as far as i know. So right now i have 2Gb on my fastest and smallest! sdd and it gets bigger and bigger. Modern PCs with fast small NVMEs for Win and large storage hdd and/or sdd are the norm. So please change that also.

**Comment by WhiteCat6142** [comment]
@Firescrat
-datadir=<dir>	Specify data directory

See following:
https://gridcoin.us/Dev/daemon-cmd.htm

**Comment by aladar42** [comment]
The process is actually very straightforward and possibly easier than many other coins, however the documentation is way too long winded, confusing, and makes it seem complicated. I would propose contacting the authors of one of the Steemit guides and publishing them on gridcoin.us, after proofreading and with some minor information updates (like noting that magnitude only updates each superblock).

**Comment by jring-o** [comment]
This repo is for code issues. for future reference use the gridcoin-community repo.

https://github.com/gridcoin-community/Gridcoin-Tasks/issues

While I agree with you, UX is secondary to operation. These improvements and others will likely be made, but not until other issues are solved.

**Comment by RoboticMind** [comment]
Here is some useful data:
https://www.reddit.com/r/gridcoin/comments/7ufckm/new_users_what_do_you_find_confusing_about/

**Comment by AyrA** [comment]
**TL;DR**

- Add binaries to the releases on Github
- Add Version number somewhere to the official download page

**Details**

While we are at it, please start attaching the compiled binaries to the releases since you have them anyways. You prevent major hassles if there is ever a need to downgrade and it also serves as a history. Most users don't want to compile the source code themselves.

Also write the version number somewhere on the download page. There is no indication for users otherwise when a new version has been released. For example right at this moment [the binary for the newest version has not yet been published](https://www.reddit.com/r/gridcoin/comments/815ygy/3780_out_now_please_update_your_wallets/dv0p3hv/), [but there is a big announcement that we should update](https://steemit.com/gridcoin/@barton26/gridcoin-mandatory-update-version-3-7-8-0-released).
